tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "moduleResolution": "node",
+    "outDir": "lib",
+    "sourceMap": true,
+    "strict": true,
+    "target": "es2020",
+    "noImplicitAny": true,
+    "noImplicitThis":true
```

Comment:
```suggestion
```

these are `true` by default when we have `"strict": true`

Reply:
Thanks, read it in the phone and didn't see I could accept the commit. But it was done below.